### PR TITLE
Add kubecon blogpost

### PR DESCRIPTION
**Pull request by vihangm**
Signed-off-by: Vihang Mehta <vihang@pixielabs.ai>

### CODE REVIEW

File: content/blog/kubecon-na-2021/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- [**Data Science for Infrastructure: Observe, Understand, Automate**](https://kccncna2021.sched.com/event/lV4c/data-science-for-infrastructure-observe-understand-automate-zain-asgar-natalie-serrino-new-relic) (Friday 10/15, 11:00am @ Concourse Hall 152)\
+Learn how Pixie's data can be transformed to high-level signals that can be used in a variety of use cases from Zain Asgar and Natalie Serrino (core contributors).
+
+Here are some of the events we'll be attending. We hope to see you at them, along with the many other exciting talks at Kubecon!
+
+**Wednesday, October 13**
+- 11:00 am: [Cloud Native and Observability Panel: The State of the Union](https://kccncna2021.sched.com/event/lV1N/cloud-native-and-kubernetes-observability-panel-the-state-of-union-bartek-plotka-red-hat-liz-fong-jones-honeycomb-josh-suereth-google-frederic-branczyk-polar-signals-rags-srinivas-infoq) - Bartek Plotka, Red Hat; Liz Fong-Jones, Honeycomb; Josh Suereth, Google; Frederic Branczyk, Polar Signals; Rags Srinivas, InfoQ
```
Comment (aimichelle) [inline content/blog/kubecon-na-2021/index.md:28]: How the people are listed for this talk is a slightly different format than the other talks.

File: content/blog/kubecon-na-2021/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+authors: ['Zain Asgar', 'Michelle Nguyen']
+emails: ['zasgar@pixielabs.ai', 'michelle@pixielabs.ai']
+featured: true
+---
+Pixie will be attending this year's Kubecon + Cloud NativeCon NA in Los Angeles! Pixie was accepted to CNCF’s Sandbox in June 2021 and we have just officially completed the [Sandbox onboarding process](https://github.com/cncf/toc/issues/674). As a newly fledged CNCF Sandbox project, we're excited to connect with the community.
+
+Come chat observability, eBPF, open-source, and more with Pixie’s core contributors at our booth in the [Project Pavilion](https://events.linuxfoundation.org/wp-content/uploads/2021/10/KubeCon_NA_2021_Collateral_17x11_v2.pdf) from 10:30am-4:30pm, on all event days.
+
+We will also be holding three events this year:
+- [**Pixie Happy Hour**](https://pixiehh.eventbrite.com/) (Tuesday 10/12, 8:30pm @ Spire 73)\
```
Comment (aimichelle) [inline content/blog/kubecon-na-2021/index.md:16]: The `/` at the end of these looks weird.

File: content/blog/kubecon-na-2021/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Discuss Pixie, eBPF, and Kubernetes over appetizers, drinks, and sweeping views of Los Angeles from Spire 73, the tallest open-air bar in the Western hemisphere.
+
```
Comment (aimichelle) [inline content/blog/kubecon-na-2021/index.md:18]: I don't think we actually need the newline between these points... Looks too spacious.

File: content/blog/kubecon-na-2021/index.md (hunk 2 above)
Comment (vihangm) [reply content/blog/kubecon-na-2021/index.md]: that was supposed to be line breaks but looks like our MD parser doesn't need it nor respect it. will fix.

File: content/blog/kubecon-na-2021/index.md (hunk 3 above)
Comment (vihangm) [reply content/blog/kubecon-na-2021/index.md]: agreed. will fix.

File: content/blog/kubecon-na-2021/index.md (hunk 1 above)
Comment (vihangm) [reply content/blog/kubecon-na-2021/index.md]: good catch.

**Pull request merged**